src/backend/joanie/tests/core/test_serializers_target_course.py

```diff
@@ -49,7 +49,7 @@ def test_serializer_target_course_get_course_runs_if_product_relation_course_run
         factories.CourseRunFactory.create_batch(2, course=target_course)
         product = factories.ProductFactory(target_courses=[target_course])
 
-        with self.assertNumQueries(4):
+        with self.assertNumQueries(6):
```

Comment:
I didn't get why this increases if we can select_related?

Reply:
Because the optimization is made at the ViewSet level that is not implied within this test

Reply:
Yep.  I think we will have to come back on all these queries optimization once we have @carofun's demo catalog with mass objects.